doc/examples/gui_disappearing_info_box.rst

```diff
@@ -0,0 +1,16 @@
+:orphan:
+
+.. _gui_disappearing_info_box:
+
+Disappearing Info Box
+==============
+
+For an introduction the GUI system, see :ref:`gui_concepts`.
+
+This example shows how to display an info box that will disappearing after a given
+time using the :class:`arcade.gui.UIDisappearingInfoBox` class.
+
+.. image:: gui_disappearing_info_box.png
```

Comment:
1. This image should probably be moved to after the heading.
2. You should use a literal include to list the source here. See the example linked below.

https://github.com/pythonarcade/arcade/blob/development/doc/examples/gui_scrollable_text.rst#L15-L17